app/src/main/java/com/klinker/android/twitter/activities/compose/Compose.java

```diff
@@ -763,16 +772,17 @@ private void startUcrop(Uri sourceUri) {
     public static final int PWICCER = 420;
 
     public boolean pwiccer = false;
+    private boolean multiTweets = true;
```

Comment:
I don't think there really needs to be a setting to disable/enable this. I actually think it is much better to allow the user to be able to select from the different options, at runtime